CMSgov/mdct-mcr
# MDCT-477 save data to db

## Description
[MDCT-477](https://qmacbis.atlassian.net/browse/MDCT-477) Report now saves data to the database! It also includes the ability to get data. We don't currently have a place in the app where it fetches and displays data, but that is related to the hydration ticket.

### How to test
- `./dev local`
- Login as a state user
- Navigate to `/mcpar/program-information/point-of-contact`
- Fill out the form and click `Save & continue`
- Open a new terminal tab and start DynamoDB admin (see README)
- Go to `localhost:8001` and view the tables where the data is saved!
- Can also fill out Section A - BSS Entities and see that get saved.

DB should save data, consolidate new and existing data, and overwrite data with new values.

### Changed Dependencies
Update your root `.env` file with

```
REPORT_TABLE_NAME=local-reports
REPORT_STATUS_TABLE_NAME=local-reports-status
```

## Code author checklist

## Reviewer checklist (two different people)
